actions/templates/actions/actions.html

```diff
@@ -101,6 +108,12 @@ <h3><b>#{{ action.id }}</b> {{ action.name }}</h3>
     {% if action.get_log_url %}
     <p><a class="text-decoration-none text-center spacing-hover-short hvr-icon-bounce" href="{{ action.log.url }}"><i class="fas fa-file-download icon-menu hvr-icon"></i> Download log</a></p>
     {% endif %}
+
+    {% if action.status == 2 or action.status == 3 %}
+    <a class="btn btn-danger hvr-sweep-to-right-danger" id="delete_log_{{ action.id }}" href="{% url 'delete_action_log' action.id %}" role="button"><i class="fas fa-trash"></i> Delete Log</a>
+    {% else %}
+    <a class="btn btn-danger disabled hvr-sweep-to-right-danger" id="delete_log_{{ action.id }}" href="{% url 'delete_action_log' action.id %}" role="button" aria-disabled="true"><i class="fas fa-trash"></i> Delete Log</a>
```

Comment:
Bootstrap link does not disable keyboard.

Reply:
That's mean we need to change the 'href' link when the button is disable to?